### PR TITLE
Feature - Implement shared component for admin page

### PR DESCRIPTION
**Pull request by anhpn0712**
#8 
e gửi code, mọi người review giúp e ạ

### CODE REVIEW

File: app/assets/stylesheets/style/css/vendor/jquery-jvectormap-1.2.2.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: center;
+}
+
+.jvectormap-zoomin {
+    top: 10px;
+}
+
+.jvectormap-zoomout {
+    top: 30px;
+}
```
Comment (khoa250200) [inline app/assets/stylesheets/style/css/vendor/jquery-jvectormap-1.2.2.css:36]: missing new empty line here

File: app/views/layouts/admin/application.html.erb [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+  <body class="loading" data-layout-config='{"leftSideBarTheme":"dark","layoutBoxed":false, "leftSidebarCondensed":false, "leftSidebarScrollable":false,"darkMode":false, "showRightSidebarOnStart": true}'>
+    <div class="wrapper">
+      <%= render partial: '/shared/partials/navbar'%>
+      <%= render partial: '/shared/partials/left_menu'%>
+      <%= render partial: '/shared/partials/footer'%>
+    </div>
     <%= yield %>
   </body>
-</html>
+</html>
```
Comment (khoa250200) [inline app/views/layouts/admin/application.html.erb:20]: should have an empty line here

File: app/views/shared/partials/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+              <h5 class="m-0 font-14">Jacob Deo</h5>
+              <span class="font-12 mb-0">Developer</span>
+            </div>
+          </div>
+        </a>
+      </div>
+    </div>
+  </div>
+</div>
+<!-- end Topbar -->
```
Comment (khoa250200) [inline app/views/shared/partials/_navbar.html.erb:206]: should have an empty line here

**Comment by anhpn0712** [comment]
- Implement shared component for admin page 

   - Implement views/shared/partials/navbar (Thắm)

   - Implement views/shared/partials/left_menu (Khoa + Ngọc Anh)
Implement views/shared/partials/footer

**Review by khoa250200 — APPROVED** [review]

**Review by hongthamnguyen2k — APPROVED** [review]

**Pull request merged**